package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "linter-jscs",
   "version": "3.3.0",
-  "main": "index.js",
+  "main": "src/linter-jscs",
   "description": "Linter plugin for JavaScript, using JSCS",
   "repository": "https://github.com/AtomLinter/linter-jscs",
   "license": "MIT",
   "engines": {
     "atom": ">=1.0.0"
   },
+  "scripts": {
+    "test": "fsmonitor -d src -s -q apm test"
```

Comment:
Does this require [fsmonitor](https://www.npmjs.com/package/fsmonitor)? If so it should be added to the `devDependencies`.